dashboard/app/views/layouts/application.html.haml

```diff
@@ -6,9 +6,6 @@
     = render partial: 'i18n/crowdin_in_context_tool'
     = render inline: File.read(Rails.root.join('..', 'shared', 'haml', 'onetrust_cookie_scripts.haml')), type: :haml, locals: {dashboard: true}
     -# Setup variables used by the Offline feature.
-    :javascript
```

Comment:
Intentional?

Reply:
Did not mean to commit! Will remove diff.